index.html

```diff
@@ -1,151 +1,252 @@
+          <li><img src="./src/assets/images/github.svg" alt="Github" /></li>
+          <li>
+            <img src="./src/assets/images/angellist.svg" alt="Angellist" />
+          </li>
+        </ul>
+      </div>
     </header>
-    <section class="wrap">
-        <h1>I am TEEJAY! Happy to have you here today!!</h1>
-        <p>I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.</p>
-        <a href="#">LET'S CONNECT</a>
-        <ul class="social">
-            <li><a href=""><img src="../src/assets/images/Vector.svg" alt="Facebook" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Linkedin.svg" alt="Linkedin" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Vector_1.svg" alt="Instagram" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Vector_2.svg" alt="Twitter" width="24" height="20"></a></li>
-            <li><a href=""><img src="../src/assets/images/Vector_3.svg" alt="Tiktok" width="24" height="20"></a></li>
+    <section class="main-grid">
+      <!-- Card 1 -->
+      <div class="cards">
+        <div class="card-img-block">
+          <img src="./src/assets/images/tonic.svg" alt="tonic" />
+        </div>
+        <div class="card-text-block">
+          <h2>Tonic</h2>
+          <ul class="list-items">
+            <li class="count">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="card-tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="card-btn" type="submit">See Project</button>
+        </div>
+      </div>
+      <!-- Card 2 -->
+      <div class="cards">
+        <div class="card-img-block order-2">
+          <img src="./src/assets/images/stories.svg" alt="tonic" />
+        </div>
+        <div class="card-text-block order-1">
+          <h2>Multi-Post Stories</h2>
+          <ul class="list-items">
+            <li class="count">FACEBOOK</li>
+            <li>Full Stack Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            Experimental content creation feature that allows users to add to an
+            existing story over the course of a day without spamming their
+            friends.
+          </p>
+          <ul class="card-tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="card-btn" type="submit">See Project</button>
+        </div>
+      </div>
+      <!-- Card 3 -->
+      <div class="cards">
```

Comment:
Kindly, as the previous reviewer recommended, I recommend you use the Article tag instead of "div class='card'"